projects/plugins/jetpack/modules/search/instant-search/components/product-ratings.jsx

```diff
@@ -21,27 +21,28 @@ import './product-ratings.scss';
  * @param {object} props - Component properties.
  * @param {number} props.count - Number of ratings.
  * @param {number} props.rating - Average rating out of five.
+ * @param {string} props.permalink - Permalink URL to product page.
  * @returns {object} Product rating component.
  */
-export default function ProductRatings( { rating = 0, count = 0 } ) {
+export default function ProductRatings( { rating = 0, count = 0, permalink } ) {
 	return (
 		<div className="jetpack-instant-search__product-rating">
 			<span aria-hidden className="jetpack-instant-search__product-rating-stars">
 				{ Array( 5 )
 					.fill( <Gridicon size={ 16 } icon="star-outline" /> )
 					.fill( <Gridicon size={ 16 } icon="star" />, 0, rating ) }
 			</span>{ ' ' }
-			<span
+			<a
 				aria-hidden
 				className="jetpack-instant-search__product-rating-count"
-				title={ sprintf(
+				href={ permalink + '#reviews' }
+			>
+				{ sprintf(
 					/* Translators: the placeholder is the number of product reviews. */
-					_n( '(%s customer review)', '(%s customer reviews)', count, 'jetpack' ),
+					_n( '%s review', '%s reviews', count, 'jetpack' ),
```

Comment:
Is it better to use %s in "% review" which would be compatible even the count is not a number.

Reply:
@kangzj do you mean using `%s` rather than `%d`?

Reply:
Fixed in 7196a1e 👍